dissect/target/plugins/apps/remoteaccess/anydesk.py

```diff
@@ -11,30 +11,43 @@
 class AnydeskPlugin(RemoteAccessPlugin):
     """
     Anydesk plugin.
+    
+    Trace file location based on https://support.anydesk.com/knowledge/trace-files#trace-file-locations
```

Comment:
```suggestion
```
You already list the URL under references down bellow, I don't think it's necessary to list it here too.

Reply:
Agreed. Both should be fixed.